netlogo-headless/src/main/workspace/AbstractWorkspace.scala

```diff
@@ -221,6 +238,21 @@ object AbstractWorkspaceTraits {
     def exportDrawingToCSV(writer:PrintWriter)
     def exportOutputAreaToCSV(writer:PrintWriter)
 
+    def checkPlotUpdates() {
+      import ExportPlotWarningAction._
+      exportPlotWarningAction match {
+        case Warn => {
+          setExportPlotWarningAction(ExportPlotWarningAction.Ignore)
+          throw new Exception("Enable plot updating to use export-plot, export-all-plots, export-world, or export-interface.")
```

Comment:
Probably move to I18N string, merged with the other similar usages.

Reply:
Done